Touch Bar Simulator/AppDelegate.swift

```diff
@@ -87,6 +92,8 @@ extension AppDelegate: NSMenuDelegate {
 
 		menu.addItem(NSMenuItem("Hide and Show Automatically").bindState(to: .dockBehavior))
 
+		menu.addItem(NSMenuItem("Open at Login").bindState(to: .openAtLogin))
```

Comment:
`Launch` is more commonly used for this.

```suggestion
		menu.addItem(NSMenuItem("Launch at Login").bindState(to: .launchAtLogin))
```